packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -341,9 +344,9 @@ function WorklistDataGrid(props) {
     if (onColumnSelect) {
       onColumnSelect(columnId);
     }
-  };
+  }, [hasSelectableRows, intl, onColumnSelect]);
```

Comment:
why is the intl object being added here?

Reply:
Because it is a dependency and should be required.